aip/general/0157.md

```diff
@@ -14,28 +14,27 @@ needs to give the user control over which fields it sends back.
 
 ## Guidance
 
-APIs **may** support partial responses in one of the three ways:
+APIs **may** support partial responses in one of two ways:
 
 ### Field masks parameter
 
 Field masks (`google.protobuf.FieldMask`) can be used for granting the user
-fine-grained control over what fields are returned. It is generally the
-recommended way for an API to support partial responses.
+fine-grained control over what fields are returned. An API **should** support the mask in a side channel.
+For example, the parameter can be specified either using an HTTP query
+parameter, an HTTP header, or a [gRPC metadata entry][1].
 
-To enable field masks, an API **should** support the masks as a [system parameter][0].
-This parameter can be specified either using an HTTP query
-parameter or an HTTP header (or a [gRPC metadata entry][1]).
+Field masks **should not** be specified in the [request](./0157.md#read-masks-as-a-request-field).
 
 - The value of the field mask parameter **must** be a `google.protobuf.FieldMask`.
-- The field mask parameter **should** be specified using `$field` as a query parameter or `X-Goog-FieldMask` as an HTTP header.
 - The field mask parameter **must** be optional:
   - An explicit value of `"*"` **should** be supported, and **must** return all
     fields.
-  - If the field mask parameter is not provided, all fields **must** be
-    returned.
+  - If the field mask parameter is omitted, it **must** default to `"*"`, unless otherwise documented.
 - An API **may** allow read masks with non-terminal repeated fields (unlike
   update masks), but is not obligated to do so.
 
+For example, Google Cloud APIs specify field masks as a [system parameter][0].
```

Comment:
I'd probably move this to the end of the first paragraph (line 24), as that's what it's an example of.